Library/Formula/plr.rb

```diff
@@ -0,0 +1,39 @@
+class Plr < Formula
+  desc "PL/R - R Procedural Language for PostgreSQL"
+  homepage "http://www.joeconway.com/plr/"
+  url "https://github.com/jconway/plr/archive/REL8_3_0_16.tar.gz"
+  sha256 "57e2384f7b51328c9e6d92a40039cae7ac3e187ece03a1d33985b751f24bfe18"
+  head "https://github.com/jconway/plr.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "postgresql"
+  depends_on "r"
```

Comment:
`r` lives in [Homebrew/science](https://github.com/Homebrew/homebrew-science/blob/master/r.rb) so if this needs that this formula needs to live there also.